Codeinwp/otter-blocks
# Icon Enhancement

Closes #802.


### Summary


- Add responsive alignment
- Change font-size to 32px as default
- New size picker for Icon Size

### Screenshots 

<image width="170" src="https://user-images.githubusercontent.com/17597852/187419148-4dfeb6e3-32aa-49e7-8f7c-54ea3d33fdc9.png" />


----

### Test instructions


1. Add FA Icons block
2. Play with the settings.
3. Add a Countdown block and do the same (they share a component, we need to make sure that it is working in both)


#### Query
```javascript
new QueryQA().select('blocks').where({ names: ['icon-alignment-and-responsive'] }).run()
```


---- 

### Checklist before the final review
